src/NuGet.Core/NuGet.Credentials/NuGet.Credentials.csproj

```diff
@@ -46,6 +46,18 @@
     </Compile>
   </ItemGroup>
 
+  <ItemGroup Condition="$(DefineConstants.Contains(SIGNED_BUILD))">
+    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
```

Comment:
We already have this declared predominantly in AssemblyInfo.cs files all over the place.

What's the benefit of declaring it here?
Should the others be converted to this approach?
Should this be converted to the AssemblyInfo.cs approach?

Reply:
In SDK based projects the assembly info is autogenerated from the csproj itself, so for SDK based projects, I'd prefer adding them in the project file. 
Not all of our projects have AssemblyInfo, so that means me lean towards the csproj file. 

I don't think there's a benefit doing it either way, it's a different way to express it. 
I'll create a follow up issue to address the inconsistency.

Reply:
https://github.com/NuGet/Home/issues/8702